[DEBUG] 2020-11-12 14:31:22,441 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:32:22,570 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:32:22,882 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:33:22,957 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:33:23,100 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:34:23,174 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:34:23,351 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:35:23,422 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:35:23,614 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:36:23,697 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:36:23,993 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:37:24,068 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:37:24,232 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:38:24,305 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:38:24,474 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:39:24,552 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:39:24,754 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:40:24,828 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:40:25,031 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:41:25,108 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:41:25,291 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:42:25,365 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:42:25,575 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:43:25,651 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:43:25,789 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:44:25,864 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:44:26,139 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:45:26,223 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:45:26,400 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:46:26,477 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:46:26,619 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:47:26,687 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:47:26,850 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:48:26,922 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:48:27,183 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:49:27,252 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:49:27,415 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:50:27,490 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:50:27,667 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:51:27,734 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:51:27,874 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:52:27,953 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:52:28,121 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:53:28,200 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:53:28,364 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:54:28,437 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:54:28,564 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:55:28,639 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:55:28,800 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:56:28,880 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:56:29,054 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:57:29,129 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:57:29,275 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:58:29,349 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:58:29,531 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 14:59:29,591 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 14:59:29,723 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:00:29,786 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:00:29,932 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:01:29,990 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:01:30,108 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:02:30,205 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:02:30,517 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:03:30,587 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:03:30,729 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:04:30,787 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:04:30,933 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:05:31,000 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:05:31,145 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:06:31,207 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:06:31,325 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:07:31,388 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:07:31,533 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:08:31,601 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:08:31,837 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:09:31,902 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:09:32,020 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:10:32,087 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:10:32,270 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:11:32,348 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:11:32,509 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:12:32,569 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:12:32,721 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:13:32,779 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:13:32,888 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:14:32,948 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:14:33,087 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:15:33,147 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:15:33,262 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:16:33,338 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:16:33,554 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:17:33,620 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:17:33,746 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:18:33,807 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:18:33,957 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:19:34,013 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:19:34,133 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:20:34,193 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:20:34,328 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:21:34,398 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:21:34,563 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:22:34,625 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:22:34,729 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:23:34,787 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:23:34,888 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:24:34,962 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:24:35,129 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:25:35,206 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:25:35,315 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:26:35,383 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:26:35,519 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:27:35,587 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:27:35,899 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:28:35,965 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:28:36,152 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:29:36,214 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:29:36,322 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:30:36,405 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:30:36,587 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:31:36,659 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:31:36,782 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:32:36,838 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:32:36,991 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:33:37,058 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:33:37,241 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:34:37,298 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:34:37,403 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:35:37,459 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:35:37,579 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:36:37,639 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:36:37,790 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:37:37,851 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:37:37,966 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:38:38,024 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:38:38,098 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:39:38,173 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:39:38,304 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:40:38,365 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:40:38,547 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:41:38,621 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:41:38,768 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:42:38,834 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:42:38,977 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:43:39,049 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:43:39,235 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:44:39,293 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:44:39,468 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:45:39,532 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:45:39,666 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:46:39,744 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:46:40,011 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:47:40,069 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:47:40,167 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:48:40,236 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:48:40,392 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:49:40,459 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:49:40,596 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:50:40,666 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:50:40,774 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:51:40,838 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:51:40,985 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:52:41,054 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:52:41,282 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:53:41,341 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:53:41,438 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:54:41,506 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:54:41,629 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:55:41,686 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:55:41,790 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:56:41,847 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:56:41,970 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:57:42,036 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:57:42,180 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:58:42,250 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:58:42,386 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 15:59:42,455 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 15:59:42,570 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:00:42,634 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:00:43,023 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:01:43,096 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:01:43,245 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:02:43,322 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:02:43,522 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:03:43,587 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:03:43,773 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:04:43,843 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:04:44,053 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:05:44,125 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:05:44,295 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:06:44,356 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:06:44,503 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:07:44,570 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:07:44,726 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:08:44,789 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:08:45,043 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:09:45,105 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:09:45,222 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:10:45,287 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:10:45,788 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:11:45,850 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:11:45,961 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:12:46,024 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:12:46,198 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:13:46,265 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:13:46,400 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:14:46,464 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:14:46,604 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:15:46,682 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:15:46,867 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:16:46,930 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:16:47,132 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:17:47,195 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:17:47,321 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:18:47,383 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:18:47,513 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:19:47,584 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:19:47,739 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:20:47,804 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:20:47,996 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:21:48,064 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:21:48,207 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:22:48,279 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:22:48,403 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:23:48,467 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:23:48,578 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:24:48,646 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:24:48,789 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:25:48,857 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:25:49,017 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:26:49,089 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:26:49,212 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:27:49,271 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:27:49,396 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:28:49,458 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:28:49,611 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:29:49,671 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:29:49,774 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:30:49,847 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:30:50,010 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:31:50,086 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:31:50,257 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:32:50,318 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:32:50,512 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:33:50,574 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:33:50,699 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:34:50,769 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:34:50,910 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:35:50,975 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:35:51,126 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:36:51,193 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:36:51,387 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:37:51,447 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:37:51,584 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:38:51,642 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:38:51,771 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:39:51,840 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:39:51,991 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 16:40:39,458 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 16:40:39,634 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:40:39.459446-127.0.0.1-862
[DEBUG] 2020-11-12 16:40:52,058 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:40:52,506 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 16:41:38,724 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 16:41:38,727 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:41:38.724708-127.0.0.1-036
[DEBUG] 2020-11-12 16:41:52,576 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:41:52,710 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 16:42:14,519 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "frappe.realtime.get_user_info"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: get_user_info() missing 1 required positional argument: 'sid'
[ERROR] 2020-11-12 16:42:14,528 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "frappe.realtime.get_user_info"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: get_user_info() missing 1 required positional argument: 'sid'
[ERROR] 2020-11-12 16:42:14,707 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:42:14.522257-127.0.0.1-bfc
[ERROR] 2020-11-12 16:42:14,710 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:42:14.529024-127.0.0.1-1c2
[ERROR] 2020-11-12 16:42:33,377 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 16:42:33,379 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:42:33.377384-127.0.0.1-736
[DEBUG] 2020-11-12 16:43:11,730 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:43:11,982 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:44:12,055 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:44:12,334 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:45:12,407 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:45:12,611 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 16:45:54,551 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 16:45:54,658 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:45:54.552621-127.0.0.1-d18
[DEBUG] 2020-11-12 16:46:12,692 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:46:12,888 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:47:12,968 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:47:13,154 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 16:47:44,353 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 16:47:44,358 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 16:47:44.354237-127.0.0.1-53e
[DEBUG] 2020-11-12 16:48:13,227 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:48:13,453 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:49:13,535 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:49:13,728 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:50:13,796 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:50:13,972 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:51:14,052 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:51:14,205 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:52:14,276 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:52:14,499 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:53:14,570 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:53:14,867 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:54:14,944 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:54:15,095 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:55:15,171 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:55:15,335 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:56:15,409 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:56:15,590 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:57:15,662 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:57:15,831 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:58:15,908 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:58:16,114 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 16:59:16,192 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 16:59:16,412 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:00:16,478 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:00:16,682 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:01:16,759 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:01:16,928 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:02:17,000 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:02:17,177 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:03:17,253 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:03:17,427 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:04:17,506 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:04:17,766 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:05:17,842 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:05:18,007 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:06:18,088 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:06:18,314 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:07:18,400 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:07:18,572 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:08:18,644 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:08:18,883 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:09:18,966 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:09:19,123 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:10:19,190 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:10:19,345 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:11:19,427 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:11:19,591 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:12:19,657 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:12:19,857 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:13:19,932 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:13:20,055 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:14:20,134 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:14:20,271 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:15:20,343 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:15:20,449 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:16:20,517 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:16:20,755 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:17:20,823 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:17:20,969 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:18:21,057 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:18:21,210 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:19:21,274 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:19:21,415 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:20:21,488 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:20:21,621 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:21:21,694 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:21:21,877 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:22:21,948 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:22:22,129 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:23:22,213 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:23:22,383 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:24:22,445 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:24:22,650 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:25:22,711 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:25:22,820 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:26:22,897 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:26:23,576 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:27:23,638 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:27:23,756 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:28:23,829 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:28:24,066 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:29:24,128 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:29:24,308 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:30:24,375 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:30:24,506 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:31:24,572 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:31:24,717 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:32:05,049 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 17:32:05,051 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:32:05.049722-127.0.0.1-dec
[DEBUG] 2020-11-12 17:32:24,782 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:32:24,986 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:33:25,050 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:33:25,377 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:34:25,446 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:34:25,591 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:34:40,846 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash33",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: create_doctor() missing 7 required positional arguments: 'doctor_name', 'specialisation', 'gender', 'city', 'years_of_experience', 'about_me', and 'mobile_no'
[ERROR] 2020-11-12 17:34:40,848 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:34:40.846584-127.0.0.1-df9
[DEBUG] 2020-11-12 17:35:25,662 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:35:25,865 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:36:25,957 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:36:26,225 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:37:26,294 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:37:26,425 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:38:25,008 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "frappe.realtime.get_user_info"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
TypeError: get_user_info() missing 1 required positional argument: 'sid'
[ERROR] 2020-11-12 17:38:25,264 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:38:25.009991-127.0.0.1-bb6
[DEBUG] 2020-11-12 17:39:23,758 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:39:24,057 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:40:24,133 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:40:24,371 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:41:24,438 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:41:24,573 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:42:24,644 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:42:24,759 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:43:24,826 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:43:24,941 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:44:25,005 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:44:25,151 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:45:25,221 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:45:25,387 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:45:50,685 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 52, in execute_cmd
    raise e
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 49, in execute_cmd
    method = get_attr(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 214, in get_attr
    method = frappe.get_attr(cmd)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1055, in get_attr
    return getattr(get_module(modulename), methodname)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 836, in get_module
    return importlib.import_module(modulename)
  File "/home/anil/medley/env/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 779, in exec_module
  File "<frozen importlib._bootstrap_external>", line 916, in get_code
  File "<frozen importlib._bootstrap_external>", line 846, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136
    data = json.loads(args.data)
                               ^
IndentationError: unindent does not match any outer indentation level
[ERROR] 2020-11-12 17:45:50,810 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:45:50.686453-127.0.0.1-dc5
[ERROR] 2020-11-12 17:46:19,257 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 17:46:19,517 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:46:19.258733-127.0.0.1-742
[DEBUG] 2020-11-12 17:46:25,455 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:46:25,609 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:47:01,869 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 17:47:01,971 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:47:01.869478-127.0.0.1-8d2
[ERROR] 2020-11-12 17:47:21,697 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 17:47:21,800 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:47:21.698201-127.0.0.1-5d0
[DEBUG] 2020-11-12 17:47:25,676 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:47:25,814 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:48:25,886 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:48:26,216 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:49:26,281 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:49:26,397 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:50:26,468 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:50:26,626 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:51:26,704 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:51:26,845 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:52:26,917 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:52:27,146 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:53:27,209 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:53:27,376 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:54:27,445 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:54:27,571 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 17:55:27,641 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:55:27,811 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:55:39,468 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 17:55:39,472 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:55:39.469158-127.0.0.1-762
[ERROR] 2020-11-12 17:56:14,643 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not _dict
[ERROR] 2020-11-12 17:56:14,745 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:56:14.644469-127.0.0.1-00c
[DEBUG] 2020-11-12 17:56:27,883 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:56:28,159 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:57:12,259 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(data.decode(), parse_float=float)
UnboundLocalError: local variable 'data' referenced before assignment
[ERROR] 2020-11-12 17:57:12,360 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:57:12.259948-127.0.0.1-e2a
[DEBUG] 2020-11-12 17:57:28,236 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:57:28,363 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:57:45,041 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(data.args.decode(), parse_float=float)
UnboundLocalError: local variable 'data' referenced before assignment
[ERROR] 2020-11-12 17:57:45,145 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:57:45.041967-127.0.0.1-8c6
[ERROR] 2020-11-12 17:58:08,913 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "about_me": "expericence",
 "city": "city",
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_name": "avinash30",
 "gender": "Female",
 "mobile_no": "00000000",
 "specialisation": "mbbs",
 "years_of_experience": "3"
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.decode(), parse_float=float)
TypeError: 'NoneType' object is not callable
[ERROR] 2020-11-12 17:58:09,013 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:58:08.913681-127.0.0.1-ebc
[DEBUG] 2020-11-12 17:58:28,434 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:58:28,574 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 17:59:22,325 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash30",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 17:59:22,435 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 17:59:22.325980-127.0.0.1-ec9
[DEBUG] 2020-11-12 17:59:28,645 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 17:59:28,815 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 18:00:02,909 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash30",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 18:00:03,014 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 18:00:02.909807-127.0.0.1-4cf
[DEBUG] 2020-11-12 18:00:28,883 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:00:29,114 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:01:29,179 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:01:29,278 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:02:29,342 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:02:29,498 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 18:02:45,465 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 124, in init_request
    make_form_dict(request)
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 133, in make_form_dict
    args = json.loads(request_data)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 3 column 9 (char 34)
[ERROR] 2020-11-12 18:02:45,561 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 18:02:45.465496-127.0.0.1-703
[DEBUG] 2020-11-12 18:03:29,569 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:03:29,801 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 18:03:46,247 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash30",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 18:03:46,252 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 18:03:46.247292-127.0.0.1-6c6
[DEBUG] 2020-11-12 18:04:29,875 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:04:30,080 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:05:30,153 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:05:30,309 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:06:30,373 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:06:30,473 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:07:30,559 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:07:30,726 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 18:08:29,140 | /home/anil/medley/apps/frappe/frappe/app.py:
Site: ordermedicines
Form Dict: {
 "cmd": "medleyhealth.medleyhealth.doctype.doctor.doctor.create_doctor",
 "doctor_details": {
  "about_me": "expericence",
  "city": "city",
  "doctor_name": "avinash30",
  "gender": "Female",
  "mobile_no": "00000000",
  "specialisation": "mbbs",
  "years_of_experience": "3"
 }
}
Request Error
Traceback (most recent call last):
  File "/home/anil/medley/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/anil/medley/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/anil/medley/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/anil/medley/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/anil/medley/apps/medleyhealth/medleyhealth/medleyhealth/doctype/doctor/doctor.py", line 136, in create_doctor
    data = json.loads(args.data)
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
[ERROR] 2020-11-12 18:08:29,451 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 18:08:29.142637-127.0.0.1-f67
[DEBUG] 2020-11-12 18:08:30,807 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:08:30,977 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:09:31,052 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:09:31,228 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:10:31,295 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:10:31,441 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:11:31,510 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:11:31,730 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[ERROR] 2020-11-12 18:11:36,822 | /home/anil/medley/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-11-12 18:11:36.806593-127.0.0.1-db3
[DEBUG] 2020-11-12 18:12:31,786 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:12:31,975 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:13:32,046 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
[DEBUG] 2020-11-12 18:13:32,234 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicines
Queued events for site ordermedicines
[DEBUG] 2020-11-12 18:14:32,307 | /home/anil/medley/apps/frappe/frappe/utils/scheduler.py:
Site: ordermedicine
Access denied for site ordermedicine
